src/pybind/mgr/dashboard_v2/tox.ini

```diff
@@ -37,7 +37,7 @@ commands =
     bash ../src/vstart.sh -n
     sleep 10
     python ./bin/ceph config-key set mgr/dashboard_v2/x/server_port 9865
-    python ./bin/ceph mgr module enable dashboard_v2
+    python ./bin/ceph mgr module enable dashboard_v2 --force
```

Comment:
I've been able to fix this issue by installing the dependencies of `requirements.txt` in the `dashboard_v2` folder using both, `pip2` and `pip3`.

Reply:
I had a different error. Mine looked like a non-deterministic error to me.